docs/corpora_schema.md

```diff
@@ -34,30 +35,44 @@ schema. If it does not, it will reject the dataset with an appropriate error mes
+    This is needed for the explorer to function. When a user requests gene expression information, the explorer needs to be able to unambiguously return a
+    single value.
+*   **No PII**. Curators agree to this requirement as part of the data submission policy. However, it is not strictly enforced in our validation tooling because it
+*   is difficult for software to predict what is and is not PII. It is up to the submitter to ensure that no metadata can be personally identifiable: no names,
+*   dates of birth, specific locations, etc. There's a [list](https://docs.google.com/document/d/1sboOmbafvMh3VYjK1-3MAUt0I13UUJfkQseq8ANLPl8/edit).
 
 ### Matrix Layers
 
-The count matrix itself can exist in several forms. Generally, there is a "raw" count matrix without scaling, filtering, normalization, etc. And there is
-a "final" matrix that is used in the publication with QC and various corrections. There can also be several intermediate matrices. We require that these matrices
-have the same dimensions, so the raw count matrix should include the same cells and genes as the final.
-
-These different transformations of the count matrix are called "layers" in several file formats. Submissions to the Data Portal must contain the "raw"
-layer. Note that AnnData objects have an attribute called [raw](https://anndata.readthedocs.io/en/latest/anndata.AnnData.raw.html#anndata.AnnData.raw). This
-could be a natural place to store the raw count matrix, but as long as the locations of the layers are specified, it can go anywhere.
-
+cellxgene's data requirements are tailored to optimize data reuse. Because each assay has different characteristics, our requirements differ by assay type. In general,
+cellxgene requires submission of "raw" data suitable for computational reuse, when a standard form exists, and strongly suggests that a "final" matrix suitable for
+visualization in the explorer be included. cellxgene uses `AnnData` for data ingestion, and uses `AnnData`'s `layers` functionality to accept multiple matrix layers,
+such as "raw" and "final". This imposes some requirements on data of all assay types:
+
+*   Anndata provides a [raw](https://anndata.readthedocs.io/en/latest/anndata.AnnData.raw.html#anndata.AnnData.raw) attribute. If a raw matrix is required for an assay
+    type, it SHOULD be stored in `AnnData.raw`, but it MAY be instead stored in `AnnData.layers["raw"]`.
+*   AnnData requires that matrix layers MUST have the same dimension, so raw count matrices MUST include the same cells and genes as the final. Because it is impractical to
+    retain all barcodes in raw and final matrices, we require that cells be filtered from both. By contrast, those wishing to reuse datasets require access to raw gene expression values,
+    so we require that genes are present in both datasets. Summarizing, any cell barcodes that are removed from the data MUST be filtered from both raw and final matrices.
```

Comment:
"we" vexes me.

Because it is impractical to retain all barcodes in raw and final matrices, cells MUST be filtered from both. By contrast, those wishing to reuse datasets require access to raw gene expression values, so genes MUST be present in both datasets.